src/pages/Contributing/ReportBug/index.md

```diff
@@ -1,4 +1,19 @@
 ---
 title: 'Report a bug'
 order: 4
----
\ No newline at end of file
+---
+
+If you discover a bug when using a MOZAIC component, please tell us.  
```

Comment:
We should define a little bit what is a bug and talk about it in the context of sketch libraries too. 
Maybe also add a paragraph to tell that if you are not sure, you can ask in the channel

Reply:
I updated the file. Tell me if it's a little more convenient for you.